test/acceptance/features/environment.py

```diff
@@ -55,3 +56,13 @@ def before_scenario(_context, _scenario):
     _context.bindings = dict()
     output, code = cmd.run(f'{ctx.cli} get ns default -o jsonpath="{{.metadata.name}}"')
     assert code == 0, f"Checking connection to OS cluster by getting the 'default' project failed: {output}"
+
+
+def after_scenario(context, scenario):
+    delete_namespace=True
+    delete_namespace=os.getenv("DELETE_NAMESPACE")
+    namespace = os.getenv("TEST_NAMESPACE")
+    if (scenario.status == Status.passed) and (delete_namespace):
+        output, code = cmd.run(f"{ctx.cli} delete namespace {namespace}")
+    else:
+        assert False, f"Deleting namespace {namespace} failed since the scenario has {scenario.status}."
```

Comment:
This should not fail if the current scenario passed, just skip the cleanup with a log message.

Reply:
it causes all the tests to fail in CI